crates/cli/src/utils/mod.rs

```diff
@@ -115,8 +115,13 @@ pub fn get_provider(config: &Config) -> Result<RetryProvider> {
 
 pub fn get_executor_strategy(config: &Config) -> ExecutorStrategy {
     if config.zksync.should_compile() {
-        info!("using zksync strategy");
-        ExecutorStrategy::new_zksync()
+        if config.zksync.evm_interpreter {
+            info!("using zksync strategy (EVM-interpreter)");
+        } else {
+            info!("using zksync strategy");
+        }
```

Comment:
Do we need to log this out? User has intentionally included the flag `--evm-interpreter` the intention is already known/understood?